app/BUILD.bazel

```diff
@@ -203,20 +203,21 @@ Genrule for source files in the view_models library.
 Because each databinding library must have a unique package name and manifest, resources must be
 imported using the proper package name when building with Bazel. This genrule alters those imports
 in order to keep Gradle building.
+>>>>>>> f782cee29e8496b4742943c22a4836039d7df2ba
```

Comment:
Please remove merge artifact. Ditto elsewhere in the PR.

Reply:
Done